src/ResourceManager/AzureBackup/Commands.AzureBackup/Models/AzureBackupContainer.cs

```diff
@@ -23,24 +23,51 @@ namespace Microsoft.Azure.Commands.AzureBackup.Cmdlets
     /// <summary>
     /// Represents Azure Backup Container
     /// </summary>
-    public class AzureBackupContainer : AzureBackupVaultCmdletBase
+    public class AzureBackupContainer : AzureBackupVaultContextObject
     {
+        /// <summary>
+        /// Type of the Azure Backup container
+        /// </summary>
         public string ContainerType { get; set; }
 
+        /// <summary>
+        /// Friendly name for the Azure Backup container
+        /// </summary>
         public string FriendlyName { get; set; }
 
+        /// <summary>
+        /// Status of health of the Azure Backup container
+        /// </summary>
         public string HealthStatus { get; set; }
 
-        public string InstanceId { get; set; }
+        /// <summary>
+        /// Id of the Azure Backup Container
+        /// </summary>
+        public string ContainerId { get; set; }
```

Comment:
DataSource derives from Container object and it would also have a DataSourceId. This was achieved in BMS by using the Instance Id. But then, we would not be able to store the Container Id (if stored also as InstanceId) for this data source. That's why InstanceId was renamed to a more explicit name - ContainerId.